packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -91,17 +82,21 @@ const PatientInfo: React.FC<PatientInfoProps> = ({ patient }) => {
     }
   };
 
-  const text = (
-    <>
-      <p className={styles.tooltipPatientName}>{name}</p>
-      <p className={styles.tooltipPatientInfo}>{info}</p>
-    </>
-  );
-
   return (
     <>
-      {truncate ? (
-        <Tooltip align="bottom-left" width={100} label={text}>
+      {patientNameIsTooLong ? (
+        <Tooltip
+          align="bottom-left"
+          width={100}
+          label={
+            <>
+              <p className={styles.tooltipPatientName}>{name}</p>
+              <p className={styles.tooltipPatientInfo}>{`${parseInt(age(patient?.birthDate))}, ${getGender(
+                patient?.gender,
+              )}`}</p>
```

Comment:
I'm a bit unclear on why this extra information is included here but not, say, as a tooltip on the patient name if it's 25 characters or less.

Reply:
Yeah that is goofy. I'll fix it. Oh yeah sorry I forgot to mention this PR includes a bunch of random refactoring.

Reply:
Actually looking at it now, I think it makes sense. My only concern is that the patient name by default looks kinda clickable, when it is actually only clickable if it is a very long name.

![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/1031876/47e77e90-0429-42d9-9e4b-f87e68ae7d94)

![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/1031876/66d0c1d7-dcfe-4124-a094-1837b678fe05)